src/plugins/data_source_management/public/components/data_source_view/data_source_view.tsx

```diff
@@ -193,21 +193,9 @@ export class DataSourceView extends React.Component<DataSourceViewProps, DataSou
         panelPaddingSize="none"
         anchorPosition="downLeft"
       >
-        <EuiContextMenuPanel>
-          <EuiPanel
-            className={'dataSourceViewOuiPanel'}
-            borderRadius="none"
-            color="transparent"
-            paddingSize="s"
-          >
-            {
-              <DataSourceDropDownHeader
-                totalDataSourceCount={1}
-                application={this.props.application}
-              />
-            }
-          </EuiPanel>
-          <EuiPanel color="subdued" paddingSize="xs" borderRadius="none">
+        <DataSourceDropDownHeader totalDataSourceCount={1} application={this.props.application} />
```

Comment:
I saw in the `DataSourceSelectable`, `DataSourceDropDownHeader ` is wrapped by `EuiContextMenuPanel `, do we need the same for data source view?

Reply:
Thank you for bringing this up. I did a close look at it and i think the Drop down header should be outside of Panel as it is using the EuiPopoverTitle in the back